lib/actions/local/configure.coffee

```diff
@@ -217,14 +218,20 @@ module.exports =
 				return if not isMounted
 				umountAsync(params.target)
 		.then (configurationSchema) ->
-			denymount params.target, (cb) ->
+			dmOpts = {}
+			if process.pkg
+				# when running in a standalone pkg install, the 'denymount'
+				# executable is placed on the same folder as process.execPath
+				dmOpts.executablePath = path.join(path.dirname(process.execPath), 'denymount')
+			dmHandler = (cb) ->
 				reconfix.readConfiguration(configurationSchema, params.target)
 				.then(getConfiguration)
 				.then (answers) ->
 					if not answers.hostname
 						removeHostname(configurationSchema)
 					reconfix.writeConfiguration(configurationSchema, answers, params.target)
 				.asCallback(cb)
+			denymount params.target, dmHandler, dmOpts
```

Comment:
These changes to `configure.coffee` were incidental to this PR. There was a `denymount` patch file in the `patches` folder that could be deleted with these changes.